### ISSUE TITLE
As user I want to see definitions for "alert and warning messages"

### ISSUE THREAD

**Issue by teittla** [HSLdevcom/jore4#1128]
Virheilmoitukselle tarvitaan selitteet, näkyisi sitä klikatessa.
-> -Will come up in accessibility testing; all icons and visual markers should have a text description, if not a pop-up

![Image](https://user-images.githubusercontent.com/105207476/214777591-d7e91113-8e76-4192-8459-3c2858a05915.png)

[The design for the popup description of the temporary and alert icons ](https://www.figma.com/file/ImSTkCqQn0nhVUtMcUm41P/JORE-4.0-UX?node-id=3306%3A304013&t=SKGFTKB8h9HvOtsJ-1)(Figma)

<img width="715" alt="Screenshot 2023-01-26 at 12 35 33" src="https://user-images.githubusercontent.com/77674841/214815293-3044c0bc-2e1e-4851-8f22-a2c8e98ff3f8.png">

Implement without version history needs.

**Comment by ellinooraikaheimo** [comment]
Reflect to accesibility audit results

**Comment by juhakolari** [comment]
Added [tooltip texts that do not require version history (Figma)](https://www.figma.com/file/ImSTkCqQn0nhVUtMcUm41P/Timetables-%7C-Routes-and-lines-%7C-JORE-4.0-UX?type=design&node-id=3306%3A304013&mode=design&t=TNXHc0R7jqtDUPy5-1).

**Comment by niknii-vincit** [comment]
Are these two kinda related #1440 ?
